### PR TITLE
Updated Android OS version for Certified Devices

### PR DESCRIPTION
**Pull request by SruthiChittaluri**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/devices/teams-ip-phones.md [hunk 1]
```diff
@@ -29,69 +29,70 @@ The Microsoft Teams certification programs ensure that customers who use certifi
...
 
 ## Currently certified Teams Displays
 
 The following devices are Certified under the Microsoft Teams Displays Android Certification Program:
 
-| Device model | Firmware version | Release date  |
-|:------------------------|:-----------------|:--------------|
-|Lenovo ThinkSmart View|`CD-18781Y.201006.099`|October 2020 |
+| Device model | Firmware version | Certification end date  | | Android OS version |
+|:------------------------|:-----------------|:--------------|----------------------------------|
```
Comment (JohanFreelancer9) [inline Teams/devices/teams-ip-phones.md:84]:
```suggestion
|Device model    | Firmware version    | Certification end date | Android OS version                               |
|:---------------------------------------|:-----------------------------------------|:-----------------------------|-----------------------------------|
```

File: Teams/devices/teams-ip-phones.md [hunk 2]
```diff
@@ -29,69 +29,70 @@ The Microsoft Teams certification programs ensure that customers who use certifi
...
 ## Currently certified Teams panels
 The following devices are Certified under the Microsoft Teams panels Certification Program:
 
-| Device model| Firmware version | Release date  |                                         
-|:------------------------|:-----------------|:--------------|
-|Crestron TSS-770 | `1.003.0082` |February 2021 |
-|Crestron TSS-1070 | `1.003.0082` |February 2021 |
-|Yealink RoomPanel | `147.15.0.1` |May 2021 |
-|Logitech TAP Scheduler | `1.3.111` | December 2021 |
+| Device model| Firmware version | Certification end date || Android OS version |
```
Comment (JohanFreelancer9) [inline Teams/devices/teams-ip-phones.md:90]:
```suggestion
| Device model| Firmware version | Certification end date | Android OS version |                                         
```

File: Teams/devices/teams-ip-phones.md [hunk 3]
```diff
@@ -29,69 +29,70 @@ The Microsoft Teams certification programs ensure that customers who use certifi
...
+| Yealink VC210           | `118.15.0.14`      | August 5, 2023 | Focus room(3m x 3m) </br> Small meeting room(4.5m x 4.5m) |  9  |
 
 ## Currently certified Teams Displays
 
 The following devices are Certified under the Microsoft Teams Displays Android Certification Program:
 
-| Device model | Firmware version | Release date  |
-|:------------------------|:-----------------|:--------------|
-|Lenovo ThinkSmart View|`CD-18781Y.201006.099`|October 2020 |
+| Device model | Firmware version | Certification end date  | | Android OS version |
```
Comment (JohanFreelancer9) [inline Teams/devices/teams-ip-phones.md:83]:
```suggestion
| Device model | Firmware version | Certification end date  | Android OS version |
```

File: Teams/devices/teams-ip-phones.md (hunk 2 above)
Comment (JohanFreelancer9) [inline Teams/devices/teams-ip-phones.md:90]:
```suggestion
| Device model| Firmware version | Certification end date | Android OS version |                                         
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @SruthiChittaluri, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**